### PR TITLE
install playwright dependencies as part of cached dependencies step

### PR DESCRIPTION
**Pull request by mattcasey**
One of my builds timed out waiting for playwright to install. But we should  be able to cache it

### CODE REVIEW

File: .github/actions/install/action.yml [hunk 1]
```diff
@@ -33,4 +33,5 @@ runs:
       shell: bash
       if: steps.cache_node_modules.outputs.cache-hit != 'true'
       run: |
-        npm ci --no-audit --no-fund
\ No newline at end of file
+        npm ci --no-audit --no-fund
+        npx playwright install --with-deps
```
Comment (mattcasey) [inline .github/actions/install/action.yml:37]: trying this command based on the official example: https://playwright.dev/docs/ci-intro

**Comment by motechFR** [comment]
Looks like this worked!

I went on the CI action and re-ran it. The install deps took 30 seconds instead of 5 minutes which means it cached properly

Screenshot from 2022-09-07 22-19-18
Screenshot from 2022-09-07 22-15-31

**Comment by motechFR** [comment]
@mattcasey As an interim option, I've split the playwright install into 2 steps,

- The npm install in the cached section
- The linux deps install just before browser test exection

As per your input, also narrowed it down to only Chromium browser, hopefully this works!

**Comment by motechFR** [comment]
@mattcasey Seems like the split approach is a non starter.

We'll need to cache the system dependencies as part of our build. Will require some searching to see where they get saved

image

**Review by motechFR — APPROVED** [review]

**Pull request merged**